frontend/nuxt.config.ts

```diff
@@ -328,6 +332,22 @@ const config: NuxtConfig = {
       },
     },
   },
+  proxy: {
+    "/api/event": "http://localhost:50288", // the key is appended as the path
```

Comment:
Should this proxy configuration be changed based on `isProd` to point to plausible? 

```suggestion
    "/api/event": isProd ? "https://plausible.io" : "http://localhost:50288", // the key is appended as the path
```

Reply:
The prod link is skipped here because, like you said, it's better to proxy at the NGINX level instead of at the app level. I assume if we added an NGINX level proxy, the `isProd` would be irrelevant?

Reply:
I'm so sorry about this Dhruv, but I had the frontend infrastructure configuration incorrect in my head :slightly_frowning_face: I had written a comment saying that you were correct, that if we're going to use Nginx to proxy, then we can remove the express proxy handler in production and only use it for localhost. However, after looking at the frontend Dockerfile and the infrastructure configuration, I realised that I had conflated the API and the frontend setup. The frontend does not use a reverse proxy currently, so we'll need to proxy using the express plugin for now indeed.

In that case, we do need the change here to point it to plausible.io in production so that when the script requests `openverse.org/api/events` it gets forwarded to `plausible.io/api/events`.

One outlying question: Are we going to have plausible running in staging? If so then we might need to configure the script domains using an environment variable so that we can include the correct subdomain per deployment stage. If not then this isn't an issue, but we probably need to have another environment variable to disable it in staging.

As for Nginx, I think there is an issue _somewhere_ to add it to the frontend, in particular to free up Nuxt from needing to serve static files. I can't find the issue right now, but I'll keep searching. If I cannot find it, I'll write up the issue and mention that in addition to serving the static files, it should also include the plausible proxy and the removal of the express middleware in production from the Nuxt app.

Reply:
Added the option to set the proxy using env var in 70f0dcd.